di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -151,6 +152,11 @@ public AuthorizeHandler(
                     Map<String, Object> attributesMap =
                             generateJsonPayload(queryParamsMap.value(JSON_PAYLOAD_PARAM));
 
+                    Map<String, Object> combinedAttributeJson =
+                            generateJsonPayload(getSharedAttributes(queryParamsMap));
```

Comment:
Why do we need to return the shared attributes in the VC? I always assumed they would just be a separate thing since we have a lambda that goes and generates them upon each request to a CRI?

Reply:
I did it to demonstrate that a cri can use the shared attributes when generating a vc and include them. We could return anything from the stub though really.

Reply:
Ah okay fair enough, so its not something we would be doing for real CRI's? If so then that sounds all good 👍

Reply:
Yeah, I think it’s dependent on the real CRIs needs what they do with them.